### PR TITLE
Update Package Installation Workflow

### PR DESCRIPTION
**Pull request by mcdonnnj**
#  #

## 🗣 Description ##

This PR updates the installation of the [cisagov/cyhy-runner](https://github.com/cisagov/cyhy-runner) package. Required packages are installed as system packages instead of through `pip`. The [cisagov/cyhy-runner](https://github.com/cisagov/cyhy-runner) package itself is installed directly instead of manually downloading and extracting the tarball.

## 💭 Motivation and Context ##

Now that the [cisagov/cyhy-runner](https://github.com/cisagov/cyhy-runner) project is in-line with our modern development practices I thought it would be good to update the Ansible role. We try to avoid installing requirements with `pip` when possible unless a `venv` is used.

## 🧪 Testing ##

Automated tests pass and I deployed an nmap instance to my testing environment and confirmed that scanning works as expected.

## ✅ Checklist ##

* [x] This PR has an informative and human-readable title.
* [x] Changes are limited to a single goal - _eschew scope creep!_
* [x] All relevant type-of-change labels have been added.
* [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
* [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
* [x] Tests have been added and/or modified to cover the changes in this PR.
* [x] All new and existing tests pass.

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -1,35 +1,17 @@
...
-- name: Install docutils, since cyhy-runner needs it
-  pip:
+- name: Install system versions of the Python packages that cyhy-runner needs
+  package:
     name:
-      - docutils
+      - python3-daemon
+      - python3-docopt
+      - python3-lockfile
+      - python3-requests
```
Comment (jsf9k) [inline tasks/main.yml:10]: This is the way.

File: tasks/main.yml [hunk 2]
```diff
@@ -1,35 +1,17 @@
...
 
-- name: Install python dependencies for cyhy-runner
+- name: Install the cyhy-runner package
   pip:
-    chdir: /var/local/cyhy/runner
-    requirements: requirements.txt
+    name: https://api.github.com/repos/cisagov/cyhy-runner/tarball/develop
```
Comment (jsf9k) [inline tasks/main.yml:14]: Much simpler!

**Review by jsf9k — APPROVED** [review]
Nice work @mcdonnnj!  This is simpler and better.

**Review by dav3r — APPROVED** [review]
Excellente! 🌶️🌶️

**Pull request merged**